website/docs/tutorials/08-interacting-with-starknet/scripting.md

```diff
@@ -0,0 +1,168 @@
+# Scripting
+In order to automate a process that includes protostar operations, you may want to build a script in language of your choice.
+
+This tutorial shows a simple example of how to do such a thing using scripting in bash.
+
+We are going to write a script that builds, tests, declares, deploys, and, in the end, calls the contract. We are also going to make use of the protostar's structured output so we can use previous' commands outputs in the following ones.
+
+### File Structure
+
+First, let's create a basic protostar file structure. You can generate it by calling `protostar init`. It looks like this:
```

Comment:
Can we add link to protostar init?

Reply:
[address feedback](https://github.com/software-mansion/protostar/pull/1242/commits/977c4c633d782c864696e16ec4f28783134b7531)